packages/sitecore-jss/src/dictionary-service.ts

```diff
@@ -0,0 +1,85 @@
+import { AxiosResponse } from 'axios';
+import mcache from 'memory-cache';
+import { AxiosDataFetcher } from './data-fetcher';
+import { DictionaryPhrases, DictionaryServiceData } from './dataModels';
+import { HttpJsonFetcher } from './httpClientInterface';
+
+export type DictionaryServiceConfig = {
+  apiHost: string;
+  apiKey: string;
+  jssAppName: string;
+  /**
+   * Custom data fetcher
+   * @see HttpJsonFetcher<T>
+   */
+  dataFetcher?: HttpJsonFetcher<DictionaryServiceData>;
+  /**
+   * Enable/disable caching mechanism
+   * @default false
```

Comment:
I think we should enable by default, and bump the value to 60000. This matches the node-headless-ssr-proxy.

Reply:
In this case - yes